buffer/src/main/java/io/netty5/buffer/bytebuffer/NioBuffer.java

```diff
@@ -271,6 +271,43 @@ public void copyInto(int srcPos, Buffer dest, int destPos, int length) {
         InternalBufferUtils.copyToViaReverseLoop(this, srcPos, dest, destPos, length);
     }
 
+    @Override
+    public Buffer writeBytes(byte[] source, int srcPos, int length) {
+        if (source.length < srcPos + length) {
+            throw new ArrayIndexOutOfBoundsException();
```

Comment:
consider adding some message to explain why this diid throw.

Reply:
done